LmDataset.py

```diff
@@ -1009,6 +1009,11 @@ def _data_str_to_numpy(self, vocab, s):
     :rtype: numpy.ndarray
     """
     words = s.split()
+
+
+    word_to_idx = [vocab[word] if word in vocab else vocab['<UNK>'] for word in words]
```

Comment:
You should not use a hardcoded constant `'<UNK>'` here, but rather use `self._unknown_label`. Actually, this is exactly the code you commented out. What was wrong with it?

Reply:
This was a hack change by Julian which I just took over. I guess he wanted to have UNK in even if it was not used in training or where self._unknown_label is not set. Could you please check with him?